docs/web/examples/websocket/webskt.rpy

```diff
@@ -0,0 +1,43 @@
+from twisted.python.failure import Failure
+from twisted.web.iweb import IRequest
+from twisted.web.resource import Resource
+from twisted.web.static import File
+from twisted.web.websocket import WebSocketResource, WebSocketTransport
+from twisted.internet.task import LoopingCall
+
+
+class WebSocketDemo:
+    loop: LoopingCall | None = None
+
+    def negotiationStarted(self, transport: WebSocketTransport) -> None:
+        self.transport = transport
+        self.counter = 0
+
+    def negotiationFinished(self) -> None:
+        def heartbeat() -> None:
+            self.counter += 1
+            self.transport.sendTextMessage(f"heartbeat {self.counter}")
+
+        self.loop = LoopingCall(heartbeat)
+        self.loop.start(1.0)
+
+    def bytesMessageReceived(self, data: bytes) -> None:
+        pass
+
+    def textMessageReceived(self, data: str) -> None:
+        self.transport.sendTextMessage(f"reply to {data}")
+
+    def connectionLost(self, reason: Failure) -> None:
+        if self.loop is not None:
+            self.loop.stop()
+
+    def pongReceived(self, payload: bytes) -> None:
```

Comment:
```suggestion
    def pongReceived(self, payload: bytes) -> None:
       """
       Called when the peer sends a ping request and after
       Twisted automatically sends the pong response.
       
       You don't need to manually handle the pong response.
       
       Note that you don't need to e
       For more info about ping/pong see
       https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_servers#pings_and_pongs_the_heartbeat_of_websockets
       """
```

Reply:
There's no ping triggered in the example (and you don't need to)